static/app/components/modals/metricWidgetViewerModal.tsx

```diff
@@ -324,6 +325,11 @@ function MetricWidgetViewerModal({
           <CloseButton onClick={handleClose} />
         </Header>
         <Body>
+          <Alert type="error" showIcon>
+            {t(
+              'This widget will stop working after September 30, 2024. The Metrics beta will end and we will retire the current solution.'
```

Comment:
Personal Reminder:

@priscilawebdev update the dates with the final one, before merging this

Reply:
Oct 7th is the final one 👍